compiler/testData/codegen/box/defaultArguments/useNextParamInLambda.kt

```diff
@@ -0,0 +1,16 @@
+// IGNORE_BACKEND: JS, JS_IR
+
+fun f(
+    f1: () -> String = { f2() },
+    f2: () -> String = { "OK" }
```

Comment:
Maybe use hier non "OK" value cause it clashes with result value in ```f { result ("OK") }``` call

Reply:
Yes! Thanks. :)